tree/treeplayer/test/dataframe/dataframe_snapshot.cxx

```diff
@@ -436,6 +436,26 @@ TEST(TDFSnapshotMore, ReadWriteNestedLeaves)
    }
 }
 
+TEST(TDFSnapshotMore, Lazy)
+{
+   const auto treename = "t";
+   const auto fname0 = "lazy0.root";
+   const auto fname1 = "lazy1.root";
+   TDataFrame d(1);
+   auto v = 0U;
+   auto genf = [&v](){++v;return 42;};
+   TSnapshotOptions opts = {"RECREATE", ROOT::kZLIB, 0, 0, 99, true};
+   auto ds = d.Define("c0", genf).Snapshot<int>(treename, fname0, {"c0"}, opts);
```

Comment:
we should also test that the file is not present at this time!

Reply:
definitively, the more checks the better. A commit is coming.